Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -171,6 +171,7 @@ void OpenALStream::SetRunning(bool running)
   {
     palSourceStop(m_source);
   }
+  return true;
```

Comment:
Just something I noticed when going over it again, but shouldn't `false` be returned in the case `m_voice_context` is null? As in:

```cpp
bool XAudio2::SetRunning(bool running)
{
   if (!m_voice_context)
    return false;

   if (running)
     m_voice_context->Play();
   else
     m_voice_context->Stop();

  return true;
}
```

Seems odd to return true in the case it's not possible to actually do anything. Ditto for the XAudio2_7 SoundStream as well.